postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -191,7 +191,7 @@ def shutdown_cb():
             check,
             rate_limit=1 / collection_interval,
             run_sync=is_affirmative(config.statement_samples_config.get('run_sync', False)),
-            enabled=is_affirmative(config.statement_samples_config.get('enabled', True)),
+            enabled=is_affirmative(config.statement_samples_config.get('enabled', True) or is_affirmative(self._config.statement_activity_config.get('enabled', True))), 
```

Comment:
```
FAILED tests/test_statements.py::test_pg_stat_statements_max_warning[10000-expected_warnings1] - AttributeError: 'PostgresStatementSamples' object has no attribute '_config'
```

you'll clean this stuff up after you run the tests and stuff, but this should be `config.statement_activity_config.get('enabled', True)`